src/app/business/common/components/detail/components/forbiddenDescription.js

```diff
@@ -0,0 +1,28 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import {RoundedButton} from '@substrafoundation/substra-ui';
+import {spacingNormal} from '../../../../../../../assets/css/variables/spacing';
+
+const Span = styled('span')`
+    margin-right: ${spacingNormal};
+`;
+
+const ForbiddenDescription = ({model, setTabIndex, permissionsTabIndex}) => (
+    <Fragment>
+        <Span>
+            {`You do not have enough permissions to see this ${model}'s description.`}
+        </Span>
+        <RoundedButton onClick={() => setTabIndex(permissionsTabIndex)}>
```

Comment:
this is not the best way to handle the onClick.
With this, on each render, a new function will be created, allocating new javascript memory.
We should better use a class component here for putting this function in hoisting.

Reply:
You're absolutely right, I went too fast here. I'm fixing it right away!

Reply:
Done!